.gitignore

```diff
@@ -25,12 +25,10 @@
 !\.zuul.yaml
 
 /bazel-*
-/mobile/bazel-*
 BROWSE
 /build
 /build_*
 *.bzlc
-/ci/bazel-*
```

Comment:
same comment as elsewhere ... this _is_ the correct place to define these i think